dashboard/test/models/section_test.rb

```diff
@@ -84,19 +84,15 @@ class SectionTest < ActiveSupport::TestCase
   end
 
   test 'name is required' do
-    assert_does_not_create(Section) do
```

Comment:
The `assert_does_not_create` is meaningless, as the `new` would not persist a section even if it were valid.